pyanaconda/rescue.py

```diff
@@ -211,6 +211,9 @@ def mount_root(self, root):
             # we have to catch the possible exception, because we
             # support read-only mounting
             try:
+                log.warning("Warning: The rescue shell will trigger SELinux autorelabel "
```

Comment:
Is this warning printed in terminal, so that the user can see it?
Also, this message should be printed after the `.autorelabel` file is successfully created, it doesn't make sense to show it in case of read-only mounted fs.

Reply:
Agree about this change.